ckanext/cfpb_extrafields/plugin.py

```diff
@@ -473,3 +473,54 @@ def group_facets(self, facets_dict, group_type, package_type):
         return self._change_facets(facets_dict)
     def organization_facets(self, facets_dict, organization_type, package_type):
         return self._change_facets(facets_dict)
+
+    # Ugh, it would be nice if the plugin interfaces could be distributed to
+    # other files instead of one large https://en.wikipedia.org/wiki/God_object
```

Comment:
Agree. The situation will be improved by splitting the plugin up (if that happens).